client/packages/host/src/components/Login/LoginLayout.tsx

```diff
@@ -40,6 +40,7 @@ export const LoginLayout = ({
         flex="1 0 50%"
         sx={{
           backgroundImage: (theme: Theme) => theme.mixins.gradient.primary,
+          backgroundSize: (theme: Theme) => theme.mixins.gradient.size,
```

Comment:
```suggestion
          backgroundSize: (theme: Theme) => theme.mixins.gradient.size,
          backgroundPosition: 'center',
          backgroundRepeat: 'no-repeat',
```

The "center" makes it look better through a bigger range of window resizes.
The "no-repeat" does nothing with your theme (if "cover" is set) but is better when the "size" property is not specified (i.e. defaulting to "contain"), so makes sense to leave it on permanently. In fact, it's arguable whether this theme might actually be better just with "no-repeat" and not changing the "backgroundSize" (this makes it never overflow the box, even with extreme aspect ratios), but that's subjective and not particularly important in most cases.